giraffe/src/components/LineHoverLayer.tsx

```diff
@@ -48,8 +48,7 @@ export const LineHoverLayer: FunctionComponent<Props> = ({
 
   const points = getLineHoverPoints(
```

Comment:
quick question: we have TS to enforce order here, but since we're passing in so many parameters here, wouldn't it be safer to pass in an object that's then deconstructed on the other end? Or is the input too dynamic on the other end?

Reply:
I suggest leaving it for now. This function is only called once. An options object is a good idea in general though.

Reply:
Makes sense